tests/commonTemplates_test.go

```diff
@@ -212,4 +218,66 @@ var _ = Describe("Common templates", func() {
 			table.Entry("[test_id:4770]golden image NS", &goldenImageNS),
 		)
 	})
+
+	Context("update old templates", func() {
+		It("should remove labels from old templates", func() {
```

Comment:
I'm not sure if this test actually does anything in the default CI environment where no old labels are present.
Is there an upgrade test environment? If yes nevermind.
Maybe still fail or at least warn this test if no old templates are found, as it indicates that the case is not actually tested?